build.rs

```diff
@@ -8,6 +8,9 @@ fn main() {
 
     println!("cargo:compiler-rt={}", cwd.join("compiler-rt").display());
 
+    // Activate libm's unstable features to make full use of Nightly.
+    println!("rustc-cfg=libm/unstable");
```

Comment:
Er sorry I'm running out of time today, but I don't really have the time to basically write this PR second-hand unfortunately. This is not the right syntax for enabling the right feature here, and I'd recommend running the build locally and poking around with the output to make sure it's activated correctly.

Reply:
Oh, oops, I just assumed that the Cargo.toml syntax still applied. I'll give it a look soon.

Reply:
It seems that the correct syntax is
```rust
println!("cargo:rustc-cfg=feature=\"unstable\"");
```
I missed the `cargo:` at first, and that's my fault, but the [build scripts docs](https://doc.rust-lang.org/cargo/reference/build-scripts.html) just say `rustc-cfg=FEATURE`. which was the confusing part.

So now it's
```rust
println!("cargo:rustc-cfg=feature=\"unstable\"");
```

Which managed to trigger

```rust
#[cfg(feature = "unstable)]
compile_error!("unstable is on.");
```

when I tested it out.

The build doesn't appear to build the libm crate itself, it just directly grabs source files out of the libm source tree, so I guess we just pretend that compiler-builtins also now has a feature named `unstable`?

Reply:
Yes that'll basically be how this works, and this looks green so that's good!